app/models/project.rb

```diff
@@ -1,12 +1,24 @@
 class Project
 
-  def self.getActive
+  def self.getActive(collection_id: nil)
```

Comment:
Shouldn't it be `get_active`?

Reply:
@lucascaton , Yes, but this is an existing method and used in many places. That is why I started adding an options param.

Later, the plan is to remove this method all together